tasks/rh-sign-image/tests/mocks.sh

```diff
@@ -6,10 +6,47 @@ function internal-request() {
   echo Mock internal-request called with: $*
   echo $* >> $(workspaces.data.path)/mock_internal-request.txt
 
-  # set to async
-  /home/utils/internal-request $@ -s false
+  /home/utils/internal-request $@
 
-  # mimic the sync output
-  echo "Sync flag set to true. Waiting for the InternalRequest to be completed."
-  sleep 2
+  sleep 1
```

Comment:
do we want to extract this to something in the .github directory for other IR tasks to use? It seems pretty generic. It could be done as a followup ticket+pr

Reply:
Hmm, I'm not opposed to it, but I'm not sure where it would fit. Any suggestions? And what makes this different than other mocks we have? The IR name extraction maybe? Anyway, I would probably treat this the same way as the rest - meaning that this will be copied to other places as needed.

Maybe the `set_ir_status` function would be nice to have ready. But with how this works, it would probably have to be baked in the utils image. Which would make it opaque, so not sure if it's a good idea (and also, it's for testing, so it doesn't seem great to have it in the image).

Reply:
Yeah, I agree. Maybe we can think of a way to deduplicate when we add this to other tasks. Or we just copy it.